src/highdicom/frame.py

```diff
@@ -52,26 +52,37 @@ def encode_frame(
         Number of bits that are required to store a pixel sample
     photometric_interpretation: int
         Photometric interpretation
-    pixel_representation: int, optional
+    pixel_representation: Union[highdicom.PixelRepresentationValues, int, None], optional
         Whether pixel samples are represented as unsigned integers or
         2's complements
-    planar_configuration: int, optional
-        Whether color samples are conded by pixel (`R1G1B1R2G2B2...`) or
-        by plane (`R1R2...G1G2...B1B2...`).
+    planar_configuration: Union[highdicom.PlanarConfigurationValues, int, None], optional
+        Whether color samples are encoded by pixel (``R1G1B1R2G2B2...``) or
+        by plane (``R1R2...G1G2...B1B2...``).
 
     Returns
     -------
     bytes
-        Pixel data (potentially compressed in case of encapsulated format
-        encoding, depending on the transfer snytax)
+        Encoded pixel data (potentially compressed in case of encapsulated
+        format encoding, depending on the transfer snytax)
 
     Raises
     ------
     ValueError
         When `transfer_syntax_uid` is not supported or when
         `planar_configuration` is missing in case of a color image frame.
 
-    """
+    Note
+    ----
+    In case of color image frames, the `photometric_interpretation` parameter
+    describes the color space of the **encoded** pixel data and data may be
+    converted from RGB color space into the specified color space upon
+    encoding.  For example, the used JPEG codec convert pixels from RGB into
```

Comment:
This does quite sound right
```suggestion
    encoding.  For example, the JPEG codec converts pixels from RGB into
```